hooks/tk-nuke_actions.py

```diff
@@ -130,7 +130,21 @@ def _create_read_node(self, path, sg_publish_data):
         
         (_, ext) = os.path.splitext(path)
 
-        valid_extensions = [".png", ".jpg", ".jpeg", ".exr", ".cin", ".dpx", ".tiff", ".tif", ".mov", ".psd"]
+        valid_extensions = [".png", 
+                            ".jpg", 
+                            ".jpeg", 
+                            ".exr", 
+                            ".cin", 
+                            ".dpx", 
+                            ".tiff", 
+                            ".tif", 
+                            ".mov", 
+                            ".psd",
+                            ".tga",
+                            ".ari",
+                            ".gif",
+                            ".iff",
+                            ".png"]
 
         if ext not in valid_extensions:
```

Comment:
Maybe the extension should be lowercased? I've seen media with capital extensions.